frontend/views/components/Page.vue

```diff
@@ -183,14 +183,24 @@ $pagePaddingDesktop: 5.5rem;
   display: flex;
   align-items: center;
 
-  > span {
-    width: fit-content;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
+  & > span {
+    max-width: 100%;
+
+    &:not(:has(*)) {
+      display: block;
+      max-width: 100%;
+    }
+
+    &:not(:has(*)),
+    & > .c-header > span {
+      width: fit-content;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
```

Comment:
What does `:has(*)` mean? I've never seen this before. /cc @SebinSong your thoughts also appreciated

Reply:
`Has any child element`. Only `GroupChat` page has child elements.

Dashboard
```
<span> Group Income Contributors<span>
```
Group Chat
```
<span>
  <div>
    ...
  </div>
</span>
```

Reply:
Since this block is starting to look quite complicated, could you please document as clearly as you can, in a comment above line 186, what these changes are doing and why these different cases? Maybe also include link references to related issues.

Reply:
Let me take some time investigating the original issue in a separate branch and get back to this thread.

Reply:
@taoeffect @Silver-IT 
Just created a PR for the original issue that has less tweaks.
https://github.com/okTurtles/group-income/pull/1650/files